pkgs/top-level/all-packages.nix

```
@@ -1162,6 +1162,8 @@ with pkgs;
 
   closureInfo = callPackage ../build-support/closure-info.nix { };
 
+  sensibleUtils = callPackage ../tools/misc/sensible-utils { };
```

```suggestion
  sensible-utils = callPackage ../tools/misc/sensible-utils { };
```
that makes more sense and also auto triggers ofborg.